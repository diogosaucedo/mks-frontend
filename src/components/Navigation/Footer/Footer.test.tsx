import { render, screen } from '@testing-library/react';
import React from 'react';

import { Footer } from '.';

describe('Footer component', () => {
  it('should render and display properties correctly', () => {
    const text = 'MKS sistemas Â© Todos os direitos reservados';

    render(<Footer></Footer>);

    const content = screen.getByText(text);
    expect(content).toBeInTheDocument();
    expect(content).toHaveStyle('font-size: 12px');
  });

  it('should match snapshot', () => {
    const { asFragment } = render(<Footer></Footer>);

    expect(asFragment()).toMatchSnapshot();
  });
});
