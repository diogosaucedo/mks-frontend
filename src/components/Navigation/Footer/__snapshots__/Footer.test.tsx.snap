// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should match snapshot 1`] = `
<DocumentFragment>
  <footer
    class="sc-kDvujY hXXoCG"
    height="34px"
    width="100%"
  >
    <p
      class="sc-gswNZR PXIuz"
      font-size="12px"
    >
      MKS sistemas Â© Todos os direitos reservados
    </p>
  </footer>
</DocumentFragment>
`;
